SimCalorimetry/EcalEBTrigPrimProducers/plugins/EcalEBTrigPrimAnalyzer.cc

```diff
@@ -58,23 +110,90 @@ EcalEBTrigPrimAnalyzer::EcalEBTrigPrimAnalyzer(const edm::ParameterSet&  iConfig
     ecal_fgvb_[i]=new TH1I(title,"FGVB",10,0,10);
   }
 
-  recHits_= iConfig.getParameter<bool>("AnalyzeRecHits");
-  debug_= iConfig.getParameter<bool>("Debug");
-  rechits_labelEB_=consumes<EcalRecHitCollection>(iConfig.getParameter<edm::InputTag>("inputRecHitsEB"));
-  primToken_=consumes<EcalEBTrigPrimDigiCollection>(iConfig.getParameter<edm::InputTag>("inputTP"));
-  tokenEBdigi_=consumes<EBDigiCollection>(iConfig.getParameter<edm::InputTag>("barrelEcalDigis"));
+
+
   nEvents_=0;
 
   hTPvsTow_eta_= new TH2F("TP_vs_Tow_eta","TP vs Tow eta ; #eta(tow); #eta(tp)",50,-2.5,2.5,50,-2.5,2.5);
   hAllTPperEvt_ = new TH1F("AllTPperEvt","TP per Event; N_{TP};  ", 100, 0., 20000.);
-  hTPperEvt_ = new TH1F("TPperEvt","N_{TP} per Event; N_{TP};  ", 100, 0., 500.);
+  hTPperEvt_ = new TH1F("TPperEvt","N_{TP} per Event; N_{TP};  ", 100, 0., 20000.);
   hTP_iphiVsieta_= new TH2F("TP_iphiVsieta","TP i#phi vs i#eta ; i#eta(tp); i#phi(tp)",10,70,80,10,340,350);
   hTP_iphiVsieta_fullrange_= new TH2F("TP_iphiVsieta_fullrange","TP i#phi vs i#eta ; i#eta(tp); i#phi(tp)",200,-100,100,350,0,350);
+  h_bxNumber_ = new TH1F ("bunchCrossing", "Bunch Crossing", 16, -12, 3);
+  h_pu_ = new TH1F ("PU", "PU", 100 , 0, 300);
+  //
+  hCluTPperEvt_ = new TH1F("CluTPperEvt","N_{TP} per Event; N_{TP};  ", 10, 0., 10.);
+  h_nXtals_ = new TH1F("nXtals","Number of xTals in the cluster; N_{xtals}; ",   20, -0.5, 19.5);
+  h_etCluTP_ = new TH1F("etCluTP","E_{T}(tp);E_{T}(tp) (GeV);Count",100,0,50); 
+  //
+  /*
+  h_nClu_[0] = new TH1F("nClu0","Number of Clusters with E_{T}>0 GeV; N_{Clu};N_{ev}", 20, 0., 20.);
+  h_nClu_[1] = new TH1F("nClu05","Number of Cluster with E_{T}>0.5 GeV; N_{Clu};N_{ev}", 20, 0., 20.);
+  h_nClu_[2] = new TH1F("nClu1","Number of Clusters with E_{T}>1 GeV; N_{Clu};N_{ev}", 20, 0., 20.);
+  h_nClu_[3] = new TH1F("nClu2","Number of Clusters with E_{T}>2 GeV; N_{Clu};N_{ev}", 20, 0., 20.);
+  h_nClu_[4] = new TH1F("nClu3","Number of Clusters with E_{T}>3 GeV; N_{Clu};N_{ev}", 20, 0., 20.);
+  */
+
+  
+  h_nClu_[0] = new TH1F("nClu0","Number of Clusters with E_{T}>0 GeV; N_{Clu};N_{ev}",   100, 0., 1000.);
+  h_nClu_[1] = new TH1F("nClu05","Number of Cluster with E_{T}>0.5 GeV; N_{Clu};N_{ev}", 100, 0., 300.);
+  h_nClu_[2] = new TH1F("nClu1","Number of Clusters with E_{T}>1 GeV; N_{Clu};N_{ev}",   100, 0., 300.);
+  h_nClu_[3] = new TH1F("nClu2","Number of Clusters with E_{T}>2 GeV; N_{Clu};N_{ev}",   100, 0., 300.);
+  h_nClu_[4] = new TH1F("nClu3","Number of Clusters with E_{T}>3 GeV; N_{Clu};N_{ev}",   100, 0., 300.);
+  
+  /*
```

Comment:
Delete commented out code, or justify with a comment why you still need to keep it